configure.ac

```diff
@@ -42,21 +42,28 @@ AC_ARG_WITH([bootloader], AS_HELP_STRING([--with-bootloader=BOOTLOADER],
 if test x$bootloader = "xauto"; then
         AC_MSG_CHECKING([whether an EFI bootloader is available])
         if test -e "/usr/lib/systemd/boot/efi"; then
-                bootloader="systemd-boot"
+                if test -e "/usr/lib/shim"; then
+                    bootloader="shim-systemd-boot"
+                else
+                    bootloader="systemd-boot"
+                fi
         elif test -e "/usr/lib/goofiboot"; then
                 bootloader="goofiboot"
         # use newer goofiboot, may be dual installed
         elif test -e "/usr/lib/gummiboot"; then
                 bootloader="gummiboot"
         else
                 AC_MSG_WARN([Cannot find a valid EFI bootloader, defaulting to systemd-boot])
-                bootloader="systemd-boot"
+                bootloader="shim-systemd-boot"
```

Comment:
We will no longer use standalone `systemd-boot`, so it's not OK if we accidentally build a previous version. Ikey was going to have a single implementation (replacing `systemd-boot` with `shim-systemd-boot`) when I asked him what his approach would be.